docs/src/content/faq.mdx

```diff
@@ -37,7 +43,34 @@ If you've received the confirmation email but haven't received your NFT:
 
 The Blockfrost team actively monitors both email and Discord communications and will help resolve any issues promptly.
 
-## What are the steps to set up the Icebreaker platform after receiving the NFT?
+### How do I get access to the private channel with Icebreaker documentation?
+
+After receiving your Icebreaker NFT:
+
+- Notify the Blockfrost team that you've received your NFT
```

Comment:
It would be nice to mention that this communication happens over Discord.

Reply:
I added that detail.